changelog.md

```diff
@@ -1,3 +1,11 @@
+3.12.1
+=======
```

Comment:
Include the update to IdentityModel w/the version, as that will be included in the Id Web release